decidim-core/lib/decidim/resourceable.rb

```diff
@@ -63,12 +63,21 @@ def link_resources(resources, link_name, data = {})
         transaction do
           resource_links_from.where(name: link_name).delete_all
           Array.wrap(resources).each do |resource|
-            Decidim::ResourceLink.create!(
-              from: self,
-              to: resource,
-              name: link_name,
-              data: data
-            )
+            payload = {
+              from_type: self.class.name,
+              from_id: id,
+              to_type: resource.class.name,
+              to_id: resource.id
+            }
+            event_name = "decidim.resourceable.#{link_name}.created"
+            ActiveSupport::Notifications.instrument event_name, this: payload do
```

Comment:
Why do you need this?

Reply:
Ah, is this needed to notify the proposal that it has been linked? Did you find any problem with the events system that prevents you from using it?

Reply:
Sorry I did it this way because it is standard Rails and @oriolgual asked me to do it so: https://github.com/decidim/decidim/issues/2348#issuecomment-358309242

I didn't knew that EventManager may be used for others than notifying :(

Reply:
Aaaah I misunderstood what you were trying to do here, it's OK then 👍